coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -190,41 +190,21 @@ class MultiPartitionPlanner(partitionLocationProvider: PartitionLocationProvider
   /**
    * Gets the partition Assignment for the given plan
    */
-  private def getPartitions(logicalPlan: LogicalPlan,
-                            queryParams: PromQlQueryParams,
-                            infiniteTimeRange: Boolean = false) : Seq[PartitionAssignment] = {
-
-    //1.  Get a Seq of all Leaf node filters
-    val leafFilters = LogicalPlan.getColumnFilterGroup(logicalPlan)
+  private def getPartitions(logicalPlan: LogicalPlan) : Seq[PartitionAssignment] = {
     val nonMetricColumnSet = dataset.options.nonMetricShardColumns.toSet
-    //2. Filter from each leaf node filters to keep only nonShardKeyColumns and convert them to key value map
-    val routingKeyMap = leafFilters.map(cf => {
-      cf.filter(col => nonMetricColumnSet.contains(col.column)).map(
-        x => (x.column, x.filter.valuesStrings.head.toString)).toMap
-    })
-
-    // 3. Determine the query time range
-    val queryTimeRange = if (infiniteTimeRange) {
-      TimeRange(0, Long.MaxValue)
-    } else {
-      // 3a. Get the start and end time is ms based on the lookback, offset and the user provided start and end time
-      val (maxOffsetMs, minOffsetMs) = LogicalPlanUtils.getOffsetMillis(logicalPlan)
-        .foldLeft((Long.MinValue, Long.MaxValue)) {
-          case ((accMax, accMin), currValue) => (accMax.max(currValue), accMin.min(currValue))
-        }
-
-      val periodicSeriesTimeWithOffset = TimeRange((queryParams.startSecs * 1000) - maxOffsetMs,
-        (queryParams.endSecs * 1000) - minOffsetMs)
-      val lookBackMs = getLookBackMillis(logicalPlan).max
```

Comment:
This is something I found while writing the other PR-- the same lookback and offset are assumed for all top/nested plans.